mona_openai/mona_openai.py

```diff
@@ -72,6 +73,19 @@ def _get_mona_single_message(
     )
 
 
+def _init_mona_class(client, context_class_name, openai_endpoint_name):
+    response = client.create_openai_context_class(
+        context_class_name, openai_endpoint_name
+    )
+    if response.get("error_message", ""):
+        logging.warning(
```

Comment:
Don't you need to raise an exception or something? You are just going to continue to the next line that says the class is created successfully

Reply:
Good catch. But I'm not raising - just warning. I don't want to kill the user's process no matter what.